migrations/probo-asset-fix-and-upload

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env node
```

Comment:
Yeah, we should do that. Also - I think we should probably title this
migration to be a bit more descriptive. I think this is
"0000-get-upload-sizes-and-reencrypt' or something?

On Thu, Mar 3, 2016 at 8:57 AM, James Cole <notifications@github.com> wrote:

> In migrations/probo-asset-fix-and-upload
> <https://github.com/ProboCI/probo-asset-receiver/pull/10#discussion_r54882103>
> :
>
> > @@ -0,0 +1,104 @@
> > +#! /usr/bin/env node
>
> Can we rename the migrations to include a number indicating the order they
> need to be run? e.g. 0001-probo-asset-fix-and-upload
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ProboCI/probo-asset-receiver/pull/10/files#r54882103>.
>